saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -748,7 +748,15 @@ def is_datetime_valid(cls, date: datetime) -> bool:
         Some systems might have incorrect time that is in the future compared to Saleor.
         At the same time, we don't want to create orders that are too far in the future.
         """
-        return date < timezone.now() + timedelta(minutes=MINUTES_DIFF)
+        current_time = timezone.now()
+        future_time = current_time + timedelta(minutes=MINUTES_DIFF)
+        if not date.tzinfo:
+            raise ValidationError(
+                message="Input 'date' must be timezone-aware. "
+                "Expected format: 'YYYY-MM-DD HH:MM:SS TZ'.",
+                code=OrderBulkCreateErrorCode.INVALID,
```

Comment:
This needs to be changed as follows, as it's failing in CI currently:

```suggestion
                code=OrderBulkCreateErrorCode.INVALID.value,
```

Reply:
Done.

Reply:
Thanks!